openedx/core/djangoapps/credentials/utils.py

```diff
@@ -28,6 +29,13 @@ def get_user_credentials(user):
         log.warning('Credentials configuration is disabled.')
         return credentials
 
+    use_cache = credentials_config.is_cache_enabled
+    if use_cache:
+        cache_key = credentials_config.CACHE_KEY + '.' + user.username
```

Comment:
I am not sure about that as currently we don't have any concrete stats but the expectation is that it will minimize the http requests to credentials each time a learner loads the dashboard. Discussed with the team and we think that its reasonable to skip this for first phase and see the live results/stats before implementing this functionality.